src/Main.elm

```diff
@@ -22,7 +21,7 @@ main =
         { init = init
         , update = update
         , subscriptions = subscriptions
-        , view = viewDocument
+        , view = Page.Index.view
```

Comment:
I think we want to keep this as viewDocument and pass the page as an parameter.

Reply:
So now we have the `routeParser` (I hope these comments come in order!?) as Katja says we can match on the `page`

something like 

```elm
viewDocument : Model -> Browser.Document Msg
viewDocument model =
    case model.page of
        Index ->
            Page.Index.view model

        CaseStudy ->
            Page.CaseStudy.view model

        Resource ->
            Page.Resource.view model
```

Reply:
Nice!